File: bank-of-models/src/main/java/ro/fmi/bnk/models/SchedulerModel.java
package ro.fmi.bnk.models;

import java.math.BigDecimal;
import java.util.Date;



public class SchedulerModel {
	private Long id;
	private String toAccount;
	private String fromAccount;
	private Date nextPayment;
	private Boolean active;
	private BigDecimal amount;

	public SchedulerModel(Long id, String toAccount, String fromAccount, Date nextPayment, Boolean active,
			BigDecimal amount) {
		super();
		this.id = id;
		this.toAccount = toAccount;
		this.fromAccount = fromAccount;
		this.nextPayment = nextPayment;
		this.active = active;
		this.amount = amount;
	}

	public SchedulerModel() {
		super();
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getToAccount() {
		return toAccount;
	}

	public void setToAccount(String toAccount) {
		this.toAccount = toAccount;
	}

	public String getFromAccount() {
		return fromAccount;
	}

	public void setFromAccount(String fromAccount) {
		this.fromAccount = fromAccount;
	}

	public Date getNextPayment() {
		return nextPayment;
	}

	public void setNextPayment(Date nextPayment) {
		this.nextPayment = nextPayment;
	}

	public Boolean getActive() {
		return active;
	}

	public void setActive(Boolean active) {
		this.active = active;
	}

	public BigDecimal getAmount() {
		return amount;
	}

	public void setAmount(BigDecimal amount) {
		this.amount = amount;
	}

}

File: bank-of-web/src/main/webapp/src/app/primary-page/pay/models/schedulerModel.ts

export class SchedulerModel {

  	id:Number;
    toAccount:String;
	fromAccount:String;
	nextPayment:String;
	active:Boolean;
	amount:Number;


	
    constructor( id: Number, toAccount: string, fromAccount: string, nextPayment: Date, active: Boolean,amount:Number) {
        this.id = id;
        this.toAccount = toAccount;
        this.fromAccount = fromAccount;
        this.nextPayment =  nextPayment.getFullYear()+"-"+(nextPayment.getMonth()+1)+"-"+nextPayment.getDate()+" ";
        this.active = active;
                this.amount = amount;

    }

static fromJSONArray(array: Array<Object>): SchedulerModel[] {
        return array["data"].map(obj => new SchedulerModel(obj['id'], obj['toAccount'], obj['fromAccount'], new Date(obj['nextPayment']), obj['active'], obj['amount']));
    }
}